app/assets/stylesheets/components/_content-metadata.scss

```diff
@@ -0,0 +1,19 @@
+
+.app-c-content-metadata {
+  @include responsive-bottom-margin;
+  @include core-16;
+
+  padding-top: $gutter-half;
+  padding-bottom: $gutter-half;
+
+  border-top: 1px solid $border-colour;
+  border-bottom: 1px solid $border-colour;
+}
+
+.app-c-content-metadata__from {
+  margin-top: $gutter-half;
+
+  a {
```

Comment:
In previous components we've tended to avoid type selectors like this and replaced with a class, e.g: `.app-c-content-metadata__from-link`. Also tend to avoid nesting BEM selectors.

Reply:
I've un-nested it. I'd like to address this at some point but it means tweaking a presenter that doesn't currently support taking a class for the link_to magic so I'm leaving an a tag in the css for now with a comment.